content/en/blog/_posts/2021-11-12-are-you-ready-for-dockershim-removal/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: blog
+title: "Dockershim removal is coming. Are you ready?"
+date: 2021-11-12
+slug: are-you-ready-for-dockershim-removal
+---
+
+**Author:** Sergey Kanzhelev, Google. With reviews from Davanum Srinivas, Elana Hashman, Noah Kantrowitz, Rey Lejano.
+
+Last year we announced that Dockershim is being deprecated: [Dockershim Deprecation FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/).
+Our current plan is to remove the dockershim from Kubernetes codebase soon.
```

Comment:
non-blocking nit: align case for dockershim
```suggestion
Last year we announced that dockershim is being deprecated: [Dockershim Deprecation FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/).
Our current plan is to remove the dockershim from Kubernetes codebase soon.
```

Reply:
Non-blocking nit: Would it make sense here to link to the Google group notification that dims put out?

Reply:
I think the blog post is a better option here - it supplies much more information than the group notification and has been translated into several other languages. 

Also seconding @reylejano's comment, I would convert it from a url to a path - that will make it default to the viewers preferred language.